code/client/src/components/NFTGrid.jsx

```diff
@@ -73,22 +71,31 @@ const NFTGridItem = ({ disabled, style, styleFullView, imageWrapperStyle, imageW
   return (
     <GridItem style={fullView ? styleFullView : style} hoverable={false} onClick={() => !fullView && interactable && setFullView(true)} data-full-view={fullView}>
       {!fullView &&
-        <Row style={{ height: wrapperStyle.height || 'auto' }}>
-          <Image
-            preview={false} src={util.replaceIPFSLink(metadata?.image) || FallbackImage} fallback={FallbackImage}
-            wrapperStyle={wrapperStyle} style={{ objectFit: 'cover', width: '100%', height: '100%' }}
-          />
+        <Row>
+          <Col span={24}>
+            <Image
+              preview={false}
+              src={util.replaceIPFSLink(metadata?.image) || FallbackImage}
+              fallback={FallbackImage}
+              wrapperStyle={wrapperStyle}
+              style={{ objectFit: 'cover', width: '100%', height: isMobile ? undefined : '100%' }}
+            />
+          </Col>
         </Row>}
       {!fullView &&
         <Row justify='space-between' style={{ padding: 8 }}>
-          {metadata && <Text style={{ fontSize: 12, lineHeight: '16px' }}>{displayName}</Text>}
-          {!metadata &&
-            <Text
-              style={{ fontSize: 12 }}
-              copyable={{ text: abbrBech32ContractAddress }}
-            >{util.ellipsisAddress(abbrBech32ContractAddress)}
-            </Text>}
-          <Text style={{ fontSize: 12, lineHeight: '16px' }}>{displayBalance}</Text>
+          <Col span={12}>
```

Comment:
should not use two columns. Row is a flex box, the code was using space-between to intentionally positioning the two child nodes at beginning and end sides